view/frontend/web/template/payment/offsite-fpx-form.html

```diff
@@ -0,0 +1,92 @@
+               'id': getCode()
+               },
+               value: getCode(),
+               checked: isChecked,
+               click: selectPaymentMethod,
+               visible: isRadioButtonVisible(),
+               enable: isActive()" />
+        <label data-bind="attr: {'for': getCode()}" class="label">
+            <span data-bind="text: getTitle()"></span>
+        </label>
+        <div data-bind="visible: isSandboxOn()" class="page messages">
+            <div role="alert" class="messages">
+                <div class="message-warning warning message">
+                    <span data-bind="i18n: 'Test mode'"></span> (<a target="_blank" href="https://www.omise.co/what-are-test-keys-and-live-keys"><span data-bind="i18n: 'more info'"></span></a>)
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="payment-method-content">
+        <!-- ko foreach: getRegion('messages') -->
+        <!-- ko template: getTemplate() -->
+        <!-- /ko -->
+        <!--/ko-->
+        <div class="payment-method-billing-address">
+            <!-- ko foreach: $parent.getRegion(getBillingAddressFormName()) -->
+            <!-- ko template: getTemplate() -->
+            <!-- /ko -->
+            <!--/ko-->
+        </div>
+        <div class="checkout-agreements-block">
+            <!-- ko foreach: $parent.getRegion('before-place-order') -->
+            <!-- ko template: getTemplate() --><!-- /ko -->
+            <!--/ko-->
+        </div>
+
+        <!-- ko if: banks().length -->
+        <form class="form" data-bind="attr: {
+              id: getCode() + 'Form',
+              }">
+
+            <fieldset data-bind="attr: {class: 'fieldset payment items ' + getCode(), id: 'payment_form_' + getCode()}">
+                <div class="field required">
+                    <label for="omise_offsite_fpx_banks" class="label">
+                        <span class="title" data-bind="i18n: 'Select Bank'"></span>
+                    </label>
+                    <script>
+                        ko.components.register('banks-select-option', {
+                            template: '<span data-bind="text: name"></span>'
+                        });
+                    </script>
+                    <select class="select-list-box"
+                            name="fpx_selected_bank"
+                            style="margin-bottom:10px"
+                            size="6"
+                            data-bind="foreach: banks,
+                                value: selectedFpxBank">
+
+                        <option data-bind="attr:{value: code,
+                         class:code}, html:'<span>'+name+'</span>'">
+                        </option>
+                    </select>
+                </div>
+            </fieldset>
+        </form>
+        <div class="actions-toolbar">
+            <div class="primary">
+                <button class="action primary checkout" type="submit"
```

Comment:
can we disable this until a bank is selected? The payment will fail anyways but nice to have some client side validation

Reply:
yes, it is good to have it. I added it now 
`enable: (getCode() == isChecked() && selectedFpxBank)`